frontend/simple/views/components/Menu/MenuParent.vue

```diff
@@ -28,19 +28,6 @@ export default {
       Menu: this.config.Menu
     }
   },
-  watch: {
-    isActive: {
-      immediate: true,
-      handler (isActive) {
-        this.config.Menu.isActive = isActive
-      }
-    }
-  },
-  computed: {
-    isActive () {
-      return this.config.Menu.isActive
-    }
-  },
```

Comment:
@taoeffect This watch and computed was implemented to support a Menu being active (opened) by default but that wasn't needed so far and I doubt it will be anytime soon, so I removed the code.

That watch immediate / handler is a tricky thing, I did this Menu based on Vue Material UI [mdMenu component](https://github.com/vuematerial/vue-material/blob/dev/src/components/MdMenu/MdMenu.vue#L83) and then to understand what was going on I've read this [question on Vue forum](https://forum.vuejs.org/t/watchers-not-triggered-on-initialization/12475/2) where it explains [where that is documented](https://forum.vuejs.org/t/watchers-not-triggered-on-initialization/12475/4)